src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/Formatting/TestFiles/Expected/FormatOnPaste.cshtml

```diff
@@ -48,13 +48,13 @@
                 @foreach (var item in ViewBag.RequestOrder)
 
                 </tbody>
-
+                
             </table>
         </div>
     </div>
     <div>
-        @if (true)
-        {
+    @if (true)
+    {
```

Comment:
This change is because the document is syntactically invalid, and the Razor syntax tree doesn't parse the close brace as a C# token.